commands/timeSinceThatcher.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+    name: 'timeSinceThatcher',
```

Comment:
This has to be all lowercase as does the file name 
as index.js does not support mixed case commands in its current state

Reply:
when you say all lowercase, do you mean every variable in the code or just the name of the module?